src/libraries/System.Private.Xml/src/System/Xml/Schema/XmlSchemaComplexContentExtension.cs

```diff
@@ -2,16 +2,17 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
 namespace System.Xml.Schema
 {
     using System.Collections;
     using System.Xml.Serialization;
 
     public class XmlSchemaComplexContentExtension : XmlSchemaContent
     {
-        private XmlSchemaParticle _particle;
-        private XmlSchemaObjectCollection _attributes = new XmlSchemaObjectCollection();
-        private XmlSchemaAnyAttribute _anyAttribute;
+        private XmlSchemaParticle? _particle;
+        private XmlSchemaObjectCollection? _attributes = new XmlSchemaObjectCollection();
```

Comment:
Looks like this can be non-nullable. Same for its corresponding property getter.

Unless the `SetAttributes` should be updated to accept nullable arguments.